dipy/reconst/rumba.py

```diff
@@ -344,8 +344,8 @@ def rumba_deconv(data, kernel, n_iter=600, recon_type='smf', n_coils=1):
     $f_j; j = 1, ..., M$ is the volume fraction of the $j^{th}$ fiber
     population with an anisotropic diffusion tensor $\bold{D_j}$.
 
-    $f_{GM}$ and $f_{CSF}$ are the volume fractions and D_{GM} and D_{CSF} are
-    the mean diffusivity coefficients of GM and CSF respectively.
```

Comment:
We use more the term "diffusivities" than "diffusivity coefficients". For example, we can use the term diffusion coefficients, but as in our case these values are "apparent" values (because the diffusion coefficient is only defined for perfectly isotropic media) then we use just diffusivities...

Reply:
Fixed!